pkg/core/blockchain.go

```diff
@@ -2787,7 +2787,7 @@ func (bc *Blockchain) InitVerificationContext(ic *interop.Context, hash util.Uin
 		}
 		err := vm.IsScriptCorrect(witness.VerificationScript, nil)
 		if err != nil {
-			return fmt.Errorf("%w: %v", ErrInvalidVerification, err) //nolint:errorlint // errorlint: non-wrapping format verb for fmt.Errorf. Use `%w` to format errors
+			return fmt.Errorf("%w: %v", ErrInvalidVerificationFunction, err) //nolint:errorlint // errorlint: non-wrapping format verb for fmt.Errorf. Use `%w` to format errors
```

Comment:
My bad, now I see that this error is not about verification function, but about verification witness script. And it goes in pair with `ErrInvalidInvocation`. Thus, can we rename both of them one more time to `ErrInvalidVerificationScript` and `ErrInvalidInvocationScript` correspondingly?